pkg/controller/volume/persistentvolume/metrics/metrics.go

```diff
@@ -32,17 +33,20 @@ const (
 	pvControllerSubsystem = "pv_collector"
 
 	// Metric names.
-	totalPVKey    = "total_pv_count"
-	boundPVKey    = "bound_pv_count"
-	unboundPVKey  = "unbound_pv_count"
-	boundPVCKey   = "bound_pvc_count"
-	unboundPVCKey = "unbound_pvc_count"
+	totalPVKey           = "total_pv_count"
+	boundPVKey           = "bound_pv_count"
+	unboundPVKey         = "unbound_pv_count"
+	boundPVCKey          = "bound_pvc_count"
+	unboundPVCKey        = "unbound_pvc_count"
+	boundPVCWithVACKey   = "bound_pvc_experimental_count"   // TODO Q: Is there better name?
```

Comment:
Is there a better name? `bound_pvc_with_vac_count`?

Reply:
Why do we need "experimental" in the name?  We'll have to remove it when it is no longer experimental and it becomes a new metric name.

Reply:
> We'll have to remove it when it is no longer experimental and it becomes a new metric name.

> Can we add a new label to the existing metrics instead of new metrics?

@jsafrane @gnufied and I were discussing add labels to existing metrics only once VAC is GA, to prevent new time-series being created every time VolumeAttributesClass feature gate is flipped (because adding/removing labels breaks downstream consumers of metric). `pv_collector_bound_pvc_with_vac_count` name works for me if we stick with metrics in k/k. 

> Another question is why we need to add those metrics in k/k repo? Is it possible to add some changes for kube_persistentvolumeclaim_info in kuberentes/kube-state-metrics. 

I will look into kuberentes/kube-state-metrics metrics repo, thank you @carlory. Added metrics to k/k because it is where alpha KEP pointed, but this might be in kuberentes/kube-state-metrics instead.

Reply:
Notes from July 22 SIG Implementation meeting on ^^:

@gnufied @jsafrane @AndrewSirenko @xing-yang @msau42  were all present and aligned on the following:

1.  We will add new label to existing PV collector metric. I will edit PR Today.  All stakeholders agreed to this. @gnufied @jsafrane especially wanted some kind of metric on PV Collector. 
2. Metrics should also be added to `kuberentes/kube-state-metrics`. We are not bound by code freeze 24th deadline for that PR. @msau42 especially thinks metric should exist in kube-state-metrics.

CC @carlory